MFVNeutralino/test/TrackMover/ntuple.py

```diff
@@ -104,32 +117,33 @@
                             primary_vertices_src = cms.InputTag('goodOfflinePrimaryVertices'),
                             packed_candidates_src = cms.InputTag('packedPFCandidates'),
                             jets_src = cms.InputTag('selectedPatJets'),
-                            #muons_src = cms.InputTag('selectedPatMuons'),
-                            #electrons_src = cms.InputTag('selectedPatElectrons'),
+                            muons_src = cms.InputTag('selectedPatMuons'),
+                            electrons_src = cms.InputTag('selectedPatElectrons'),
                             track_ref_getter = jmtTrackRefGetter,
                             min_jet_pt = cms.double(0), 
                             min_jet_ntracks = cms.uint32(2), 
                             njets = cms.uint32(cfg.njets),
                             nbjets = cms.uint32(cfg.nbjets),
                             tau = cms.double(1.), #FIXME default 1.
+                            halftoss = cms.bool(True), #FIXME True for exotic-Higgs and False otherwise
                             sig_theta = cms.double(cfg.angle),
                             sig_phi = cms.double(cfg.angle),
                             )
 
     modifiedVertexSequence(process, ex, tracks_src = tracks_name,
-                           min_track_sigmadxy = 0,
-                           min_track_rescaled_sigmadxy = cfg.nsigmadxy,
-                           )
+                          min_track_sigmadxy = 0,
+                          min_track_rescaled_sigmadxy = cfg.nsigmadxy,
+                          )
 
     for x in 'mfvVerticesToJets', 'mfvVerticesAuxTmp', 'mfvVerticesAuxPresel':
-        getattr(process, x + ex).track_ref_getter.tracks_maps_srcs.append(cms.InputTag(tracks_name))
+       getattr(process, x + ex).track_ref_getter.tracks_maps_srcs.append(cms.InputTag(tracks_name))
 
     tree = cms.EDAnalyzer('MFVMovedTracksTreer',
                           jmtNtupleFiller_pset(settings.is_miniaod),
                           sel_tracks_src = cms.InputTag('mfvVertexTracks' + ex, 'all'),
                           mover_src = cms.string(tracks_name),
                           vertices_src = cms.InputTag(auxes_name),
-                          max_dist2move = cms.double(0.02),
+                          max_dist2move = cms.double(0.2), #FIXME relate to vtxunc 
```

Comment:
Not sure what this FIXME means. Maybe add more to the comment? I guess the idea is that one might want to set this differently in some circumstances?